packages/migrate/index.js

```diff
@@ -145,6 +145,7 @@ const Migrate = {
 
     try {
       global.artifacts = clone.resolver;
+      global.config = clone;
```

Comment:
Should we `normalize` the config here?

Reply:
It's already effectively done when cloning

Reply:
I know, this stuff should be more uniform... but it didn't seem worth it to make it uniform at this time